litmus-portal/cluster-agents/subscriber/pkg/k8s/operations.go

```diff
@@ -157,53 +159,48 @@ func IsClusterConfirmed() (bool, string, error) {
 }
 
 // ClusterRegister function creates litmus-portal config map in the litmus namespace
-func ClusterRegister(clusterData map[string]string) (bool, error) {
-	clientset, err := GetGenericK8sClient()
+func ClusterRegister(accessKey string) (bool, error) {
+	clientset, err:= GetGenericK8sClient()
 	if err != nil {
 		return false, err
 	}
 
-	newConfigMapData := map[string]string{
-		"IS_CLUSTER_CONFIRMED": clusterData["IS_CLUSTER_CONFIRMED"],
-		"SERVER_ADDR":          clusterData["SERVER_ADDR"],
-		"AGENT_SCOPE":          clusterData["AGENT_SCOPE"],
-		"COMPONENTS":           clusterData["COMPONENTS"],
-		"START_TIME":           clusterData["START_TIME"],
-		"VERSION":              clusterData["VERSION"],
-		"SKIP_SSL_VERIFY":      clusterData["SKIP_SSL_VERIFY"],
-		"CUSTOM_TLS_CERT":      clusterData["CUSTOM_TLS_CERT"],
+	isClusterConfirmed, _, err := IsClusterConfirmed()
```

Comment:
As mentioned by Shomo, this function is not needed here. Instead, you can directly set the value to `true` since we are already updating the map here: https://github.com/litmuschaos/litmus/blob/cf86f7e308695a2bdfbeeb585f3ab809b81dcdcd/litmus-portal/cluster-agents/subscriber/subscriber.go#L115